contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -83,16 +83,20 @@ contract TACoChildApplication is ITACoRootToChild, ITACoChildApplication, Initia
         StakingProviderInfo storage info = stakingProviderInfo[stakingProvider];
         address oldOperator = info.operator;
 
-        if (stakingProvider != address(0) && operator != oldOperator) {
-            info.operator = operator;
-            // Update operator to provider mapping
-            stakingProviderFromOperator[oldOperator] = address(0);
-            stakingProviderFromOperator[operator] = stakingProvider;
-            info.operatorConfirmed = false;
-            // TODO placeholder to notify Coordinator
+        if (stakingProvider == address(0) || operator == oldOperator) {
```

Comment:
Sorry if it's a dumb question, but why not manage this with a `require` statement?

```suggestion
require (stakingProvider != address(0) && operator != oldOperator);
```

Reply:
it's not dumb question. Because there is no real point to fail in xchain txs, it will be just skipped and that's it. But some sanity check to decrease gas consumption we want to do

Reply:
got it, thanks!